app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int = 7,
+            clean_mark: int = 1,
+            brand: str = "some_car"
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float = 1.0,
+            clean_power: int = 10,
+            average_rating: float = 5.0,
+            count_of_ratings: int = 100
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
```

Comment:
In the requirements these attributes can accept only specific values. Let's check them and if value is incorrect, set some default correct value